### ISSUE TITLE
Dataset Preparation

### ISSUE THREAD

**Issue by LoveIsAGame** [shenyunhang/DRN-WSOD-pytorch#8]
Thank you for your work! 
I try to use the new data set for training, but I don't know how to construct my data format. Can you help me?

Look forward to your reply！

**Comment by LoveIsAGame** [comment]
Can I use my own data set to construct data in coco or VOC format for training? Because this is a weakly supervised network, I don't know how it uses the data in it.

**Comment by shenyunhang** [comment]
Yes, we can use our own data set to construct data in coco format for training.

To use custom datasets, please refer to this [documents](https://github.com/shenyunhang/DRN-WSOD-pytorch/blob/DRN-WSOD/docs/tutorials/datasets.md#use-custom-datasets).

We also recommend to use [pycococreator](https://github.com/waspinator/pycococreator) to generate annotations in the format COCO requires.

**Comment by LoveIsAGame** [comment]
Thanks,i will try. And I have another problem ，The screenshot is below。
![图片](https://user-images.githubusercontent.com/49852180/103184909-f4ba4400-48f4-11eb-9744-a6e199fa4fa0.png)
![图片](https://user-images.githubusercontent.com/49852180/103184917-ff74d900-48f4-11eb-8079-0b7d9d7684f6.png)

**Comment by shenyunhang** [comment]
Hi, this problem is not directly related to DRN-WSOD-pytorch.

It should be some problem with PyTorch and CUDA.
